test/ibmq/experiment/test_experiment_data_integration.py

```diff
@@ -53,8 +53,7 @@ def setUpClass(cls):
         if not cls.provider.has_service('experiment'):
             raise SkipTest("Not authorized to use experiment service.")
 
-        # cls.backend = cls._setup_backend()  # pylint: disable=no-value-for-parameter
-        cls.backend = cls.provider.get_backend('ibmq_qasm_simulator')
```

Comment:
What about also fixing this class?

https://github.com/Qiskit/qiskit-ibmq-provider/blob/58d1ca3a9c2ffcfc6625c4483f7b113573b3d4c0/test/ibmq/experiment/test_experiment_server_integration.py#L43